manim/animation/transform.py

```diff
@@ -359,21 +498,81 @@ def begin(self) -> None:
 
 
 class FadeToColor(ApplyMethod):
+    """Animation that changes color of a mobject.
+
+    Examples
+    --------
+
+    .. manim:: FadeToColorExample
+
+        class FadeToColorExample(Scene):
+            def construct(self):
+                self.play(FadeToColor(Text("Hello World!"), color=RED))
+
+    """
+
     def __init__(self, mobject: Mobject, color: str, **kwargs) -> None:
         super().__init__(mobject.set_color, color, **kwargs)
 
 
 class ScaleInPlace(ApplyMethod):
+    """Animation that scales a mobject by a certain factor.
+
+    Examples
+    --------
+
+    .. manim:: ScaleInPlaceExample
+
+        class ScaleInPlaceExample(Scene):
+            def construct(self):
+                self.play(ScaleInPlace(Text("Hello World!"), 2))
+
+    """
+
     def __init__(self, mobject: Mobject, scale_factor: float, **kwargs) -> None:
         super().__init__(mobject.scale, scale_factor, **kwargs)
 
 
 class ShrinkToCenter(ScaleInPlace):
+    """Animation that makes a mobject shrink to center.
```

Comment:
```suggestion
    """Animation that makes an mobject shrink to center.
```

Reply:
do we spell `mobject` as 'm-object' or 'mobject' as in '[mob](https://www.google.com/search?q=mob&oq=mob&aqs=chrome..69i57j69i59j69i61j69i60l2j5.2429j0j1&sourceid=chrome&ie=UTF-8)'

Reply:
I've always heard it as an m-object, an abbreviation of a mathematical object.
I don't have any strong opinions on that, but mob-ject sounds weird.

Reply:
@Kiran-Raj-Dev please be careful with suggestions like this: at some point we agreed to spell it as "a mobject", see e.g. <https://docs.manim.community/en/stable/tutorials/building_blocks.html>.

Reply:
Oh okay, fine 👍